SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -175,7 +175,7 @@ For more details, see the [Cloud deployment workflow page]({{urlRoot}}/content/c
 
 
 <br/>------<br/>
-_2019-07-31 Page updated with limited editorial review_
-<br>_2019-06-27 Page updated with editorial review_
+_2019-08-02 Page updated with editorial review_
```

Comment:
```suggestion
_2019-07-31 Page updated with limited editorial review._</br>
```
Not sure this page is updated? And not sure any of the tech writers have added an editorial review?